data_analysis/compare_scheduled_and_rt.py

```diff
@@ -96,7 +96,7 @@ def sum_trips_by_rt_by_freq(
     rt_df: pd.DataFrame,
     sched_df: pd.DataFrame,
     agg_info: AggInfo,
-    holidays: List[str] = ["2022-05-31", "2022-07-04", "2022-09-05", "2022-11-24", "2022-12-25"]) -> pd.DataFrame:
+    holidays: List[str] = ["2022-05-30", "2022-07-04", "2022-09-05", "2022-11-24", "2022-12-25"]) -> Tuple[pd.DataFrame, pd.DataFrame]:
```

Comment:
Yeah good point, I listed them out though because I wanted to specifically only use the holidays observed by the CTA (from [here](https://www.transitchicago.com/schedules/))... Would you be ok with me just merging this and making a follow-up ticket to use the `holidays` library to generate dates based on the list observed by CTA?

Reply:
(It looks like `holidays` lets you get a date based on the name of the holiday, but I wonder about how we would handle what years to cover)

Reply:
For sure, you can go ahead and merge this. I'll look more into the `holidays` library.